doc/source/learning/overview.rst

```diff
@@ -0,0 +1,38 @@
+Overview
+=====================
+
+This part of the documentation will help new developers to understand what is Airship and all its
+ecosystem of tools.
+
+This provides links to documentation and videos to help learning, but also to as an entry point to
+find the important documentation.
+
+First let's see how the Airship parts are going to work together.
+
+
```

Comment:
I believe this should link to developer's documentation first. Explain how we do things in this repo is something ppl might be familiar, but it's something worth repeating at the start.

Reply:
I agree